lile.go

```diff
@@ -111,33 +113,78 @@ func CreateServer() *grpc.Server {
 	return gs
 }
 
-// NewTestServer is a helper function to create a gRPC server on a unix socket
-// it returns the socket location and a func to call which starts the server
-func NewTestServer(s *grpc.Server) (string, func()) {
+//   Creates a server listener dependent on the underlying platform. Windows
+// hosts will have a Windows Named pipe, anything else gets a UNIX socket
+func getTestServerTransportOrPanic()(string, net.Listener) {
```

Comment:
I think this should just be named `getTestServerTransport` and then we can panic or do something appropriate in the new test server method?

Reply:
Sure thing! I'll add it and whatever else needs to happen when this build finishes